src/stream/stream/mod.rs

```diff
@@ -372,6 +374,40 @@ extension_trait! {
             Chain::new(self, other)
         }
 
+            #[doc = r#"
+            Creates an stream which copies all of its elements.
+
+            # Examples
+
+            Basic usage:
+
+            ```
+            # fn main() { async_std::task::block_on(async {
+            #
+            use async_std::prelude::*;
+            use std::collections::VecDeque;
+
+            let v: VecDeque<_> = vec![&1, &2, &3].into_iter().collect();
```

Comment:
Could you replace `VecDeque` with `stream::from_iter`? (like https://github.com/async-rs/async-std/pull/447)